prow/plank/plank_test.go

```diff
@@ -510,6 +516,7 @@ func TestSyncKubernetesJob(t *testing.T) {
 			expectedPodHasName: true,
 			expectedNumPods:    1,
 			expectedReport:     true,
+			expectedURL:        "NEWPOD/pending",
```

Comment:
Can you follow-up on this with a separate issue? I might be able to get at it at some point if nobody else has fixed it.